### ISSUE TITLE
Improve performance on low-end laptops

### ISSUE THREAD

**Issue by Poobslag** [Poobslag/turbofat#59]
I have an older Ubuntu laptop but the game doesn't run very well on it. It pauses and stutters frequently. It'll freeze up for 2 seconds, and then rapidly catch up and replay all your inputs.

This game shouldn't have any significant hardware requirements, it should be able to run on older PCs and PCs without gaming-related graphics cards. We should figure out the performance bottlenecks and provide a "Low Detail" mode if necessary.

**Comment by Poobslag** [comment]
Someone else online, `Just A Cake`, has echoed that the game runs poorly on their laptop as well.

**Comment by Poobslag** [comment]
Experimenting on an old Ubuntu laptop, I found that `Creature.tscn` scene is the source of the performance issues.

The four creatures on the overworld reduce performance to about 7-10 FPS, so each creature seems like it takes about 25 ms to simulate. However, reducing the size of the sprite sheets drastically improves performance.:

  * I modified `creature-loader.gd` to only load the creature's ears. Even with just the ears, performance was reduced from 55 FPS to 45 FPS.
  * Reducing the sprite sheet from 3080x1026 to 1024x1024 increased performance back to the 55 FPS range.
  * Changing the sprite sheet to 4096x1024 had no effect. I'd read before that some older video cards worked better with powers of 2.

We should revisit this issue after #157.
